parquet-cli/src/main/java/org/apache/parquet/cli/BaseCommand.java

```diff
@@ -339,7 +339,7 @@ private D advance() {
                   return next;
                 } catch (IOException e) {
                   throw new RuntimeException(
-                      "Failed while reading Parquet file: " + source, e);
+                    "Failed while reading Parquet file: " + source, e);
```

Comment:
(nit) Unrelated formatting-only change.

Reply:
Reverted the change, this was due to a different style profile that was enabled.